### PR TITLE
Added gov txs to Txhistory

### PR DESCRIPTION
**Pull request by fedekunze**
Added `MsgSumbitProposal` and `MsgDeposit` to gov transactions in tx history

Component:

screen shot 2018-10-19 at 12 46 52 pm

### CODE REVIEW

File: assets/txs.js [hunk 1]
```diff
@@ -435,5 +435,135 @@ export default [
         fee: {}
       }
     }
+  },
```
Comment (NodeGuy) [inline assets/txs.js]: We don't have a top-level directory named `assets`.  Did you intend for this file to go somewhere else?

File: src/components/TmLiAnyTransaction/TmLiAnyTransaction.vue [hunk 2]
```diff
@@ -31,13 +38,20 @@ export default {
           "cosmos-sdk/BeginRedelegate"
         ].indexOf(this.type) !== -1
       )
+    },
+    governanceTx() {
+      return (
+        ["cosmos-sdk/MsgSubmitProposal", "cosmos-sdk/MsgDeposit"].indexOf(
+          this.type
+        ) !== -1
```
Comment (NodeGuy) [inline src/components/TmLiAnyTransaction/TmLiAnyTransaction.vue]:
```JavaScript
[`cosmos-sdk/MsgSubmitProposal`, `cosmos-sdk/MsgDeposit`]. includes(this.type)
```

File: src/components/TmLiGovTransaction/TmLiGovTransaction.vue [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    div(slot="details")
+      | On&nbsp;
+      router-link(:to="this.URL + '/' + tx.proposal_id") Proposal &#35;{{ tx.proposal_id }}
+</template>
+
+<script>
+import TmLiTransaction from "../TmLiTransaction/TmLiTransaction"
+import colors from "../TmLiTransaction/transaction-colors.js"
+import numeral from "numeral"
+
```
Comment (NodeGuy) [inline src/components/TmLiGovTransaction/TmLiGovTransaction.vue]: Thank you for using whitespace!

File: src/components/TmLiGovTransaction/TmLiGovTransaction.vue [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+      if (this.deposit) return colors.gov.deposit
+    }
+  },
+  methods: {
+    prettify(amount) {
+      const amountNumber = Number(amount)
+      if (Number.isInteger(amountNumber)) {
+        return numeral(amountNumber).format(`0,0`)
+      }
+      return numeral(amountNumber).format(`0,0.00`)
```
Comment (NodeGuy) [inline src/components/TmLiGovTransaction/TmLiGovTransaction.vue]:
```JavaScript
return numeral(amountNumber).format(
  Number.isInteger(amountNumber) ? `0,0` : `0,0.00`
)
```

File: src/components/TmLiGovTransaction/index.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import TmLiGovTransaction from "./TmLiGovTransaction.vue"
```
Comment (NodeGuy) [inline src/components/TmLiGovTransaction/index.js]: What's the purpose of this file?

Comment (fedekunze) [reply src/components/TmLiGovTransaction/index.js]: nothing really, there are a bunch of them so we'll have to delete them all

Comment (fedekunze) [reply src/components/TmLiGovTransaction/index.js]: I deleted only the gov one on this PR

**Comment by fedekunze** [comment]
updated description with attached screenshot cc @NodeGuy @faboweb @jbibla

**Pull request merged**